public/settings.json

```diff
@@ -3,5 +3,6 @@
   "google": {
     "clientId" : "147813579094-8qbv6canv6167qlga4k9788dg0qmgokn.apps.googleusercontent.com",
     "cookiePolicy": "single_host_origin"
-  }
-}
\ No newline at end of file
+  },
+  "clientUrl": "https://hermes.api.dev.k8s.nerds.sh/graphql"
```

Comment:
This isn't really the url of the client, right? (since we're already on the client) Let's rename this to something like `apiUrl`

Reply:
Done